packages/flutter/lib/src/widgets/layout_builder.dart

```diff
@@ -113,14 +113,18 @@ class _LayoutBuilderElement extends RenderObjectElement {
           built = widget.builder(this, constraints);
           debugWidgetBuilderValue(widget, built);
         } catch (e, stack) {
-          built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack));
+          built = ErrorWidget.builder(
+            _debugReportException(ErrorDescription('building $widget'), e, stack, this)
+          );
         }
       }
       try {
         _child = updateChild(_child, built, null);
         assert(_child != null);
       } catch (e, stack) {
-        built = ErrorWidget.builder(_debugReportException(ErrorDescription('building $widget'), e, stack));
+        built = ErrorWidget.builder(
+            _debugReportException(ErrorDescription('building $widget'), e, stack, this)
```

Comment:
trivial nit: the code is inconsistent about how much to indent the argument here (compare line 117 and line 126)